### ISSUE TITLE
Github.dev not working in portable version

### ISSUE THREAD

**Issue by uugaabuugaa** [Floorp-Projects/Floorp-Portable#35]
Getting error "Unable to resolve workspace folder (ENOPRO: No file system provider found for resource vscode-vfs: .....:" But it worked perfectly on regular Floorp. I thought it was a portable browser error so tried on portable version of Librewolf and it works there. Am I missing something?

**Comment by typeling1578** [comment]
Which platform do you use, Windows or Linux?

**Comment by uugaabuugaa** [comment]
Windows 10

**Comment by typeling1578** [comment]
We were unable to reproduce this issue.
Could you please provide detailed reproduction steps?

**Comment by uugaabuugaa** [comment]
https://github.com/user-attachments/assets/eeed4042-d6da-4d7c-a786-22b0be4a7fa4

I don't think there is any reproduction steps. I just went to the tab. Pressed `.` to get to github.dev and this is happening. Where on the other browsers I see folder tree as usual.

**Comment by uugaabuugaa** [comment]
Okay. I think I fixed it. After clicking on try again button it said something about network failed to... So I went to firewall settings and allowed everything. It still wasn't working for some reason but after turning off enhanced tracking protection it is working perfectly. Then I re-enabled enhanced protection and it's still working. Still a bit strange since with Librawolf it worked ok without me doing anything while on enhanced tracking protection as well.
